include/inspector/signal_extractor_c.h

```diff
@@ -60,12 +57,11 @@ namespace gr {
        * \param rate Desired output sample rate if resample is enabled
        * \param osf Oversampling factor of the input signal (relative to map input)
        */
-      static sptr make(int signal, bool resample = false,
-                       float rate = 1.0, float osf = 1.0);
-      virtual void set_signal(int signal) = 0;
```

Comment:
Isn't this required by the callback function?

Reply:
Yes! Missed that.

Reply:
Yes! Missed that.